blockchain/types/accountkey/account_key_test.go

```diff
@@ -162,3 +168,125 @@ func genAccountKeyRoleBased() AccountKey {
 
 	return NewAccountKeyRoleBasedWithValues(AccountKeyRoleBased{txKey, updateKey, feeKey})
 }
+
+func TestAccountKeyWeightedMultiSig_Validate(t *testing.T) {
+	// declare special block numbers and set hardForkBlockNumberConfig
+	var (
+		blockBeforeHF = uint64(4)
+		blockHF       = big.NewInt(5)
+		blockAfterHF  = uint64(6)
+	)
+	fork.SetHardForkBlockNumberConfig(&params.ChainConfig{IstanbulCompatibleBlock: blockHF})
```

Comment:
Maybe here as well? https://github.com/klaytn/klaytn/pull/997#discussion_r672855505

Reply:
@jeongkyun-oh Yes, it would be updated to use `defer`